build.gradle

```diff
@@ -18,11 +21,17 @@ buildscript {
         maven {
             url "https://plugins.gradle.org/m2/"
         }
+        maven {
+            url "https://maven.mozilla.org/maven2"
+        }
     }
     dependencies {
         classpath "com.android.tools.build:gradle:$android_gradle_plugin_version"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 
+        // For Kotlin-level metrics scaffolding generation.
+        classpath "org.mozilla.telemetry:glean-gradle-plugin:$glean_version"
```

Comment:
I know this is weird, but until mozilla-mobile/android-components#4985 is merged you should keep using the script :( Applications using Android Components should use the plug-in published from there.